bundles/org.openhab.binding.androidtv/src/main/java/org/openhab/binding/androidtv/internal/protocol/googletv/GoogleTVConnectionManager.java

```diff
@@ -465,8 +465,6 @@ private void initialize() {
         config.reconnect = (config.reconnect > 0) ? config.reconnect : DEFAULT_RECONNECT_SECONDS;
         config.heartbeat = (config.heartbeat > 0) ? config.heartbeat : DEFAULT_HEARTBEAT_SECONDS;
         config.delay = (config.delay < 0) ? 0 : config.delay;
```

Comment:
It would be better to define min and max values on the [config parameters](https://github.com/openhab/openhab-addons/blob/f24f0f0251eec789a17c6949f7dc87afd868dda6/bundles/org.openhab.binding.androidtv/src/main/resources/OH-INF/thing/thing-types.xml#L103-L134). That way UIs and the framework can validate the parameters and give users proper feeback.

See: https://www.openhab.org/docs/developer/addons/config-xml.html#xml-structure-for-configuration-descriptions